DoSomethingArchive/legacy-website
# Verify credentials against northstar

#### What's this PR do?

This PR adds "mock" authentication to Northstar after Drupal successfully authenticates a user locally.
#### How should this be manually tested?

On your local box, pull down this code and run the ol' `drush fra -y`. Log in to your account, and you _should_ be able to see your Northstar ID saved to your Drupal profile (only visible from the admin profile interface).
#### Any background context you want to provide?

This is a cautious step towards _actually_ authenticating all logins against Northstar. Let's just let it run in the background for a while and hopefully nothing goes wrong!
#### What are the relevant tickets?

References #4526.

---

For review: @angaither
